superdesk/publish/transmitters/file_output.py

```diff
@@ -8,24 +8,22 @@
 # AUTHORS and LICENSE files distributed with this source code, or
 # at https://www.sourcefabric.org/superdesk/license
 
-from superdesk.publish.publish_service import PublishService, get_file_extension
+from superdesk.publish.publish_service import PublishService
 from superdesk.publish import register_transmitter
 from superdesk.errors import PublishFileError
+from os import path
 
 errors = [PublishFileError.fileSaveError().get_error_description()]
 
 
 class FilePublishService(PublishService):
     def _transmit(self, queue_item, subscriber):
-        config = queue_item.get('destination', {}).get('config', {})
         try:
-            # use the file extension from config if it is set otherwise use extension for the format
-            extension = config.get('file_extension') or get_file_extension(queue_item)
-            with open('{}/{}-{}-{}.{}'.format(config['file_path'],
-                                              queue_item['item_id'].replace(':', '-'),
-                                              str(queue_item.get('item_version', '')),
-                                              str(queue_item.get('published_seq_num', '')),
-                                              extension), 'wb') as f:
+            config = queue_item['destination']['config']
+            file_path = config['file_path']
+            if not path.isabs(file_path):
+                file_path = "/" + file_path
+            with open(path.join(file_path, '{}'.format(PublishService.get_filename(queue_item))), 'wb') as f:
```

Comment:
oh right, I've changed this one and forgot to remove this format, it's useless here indeed, thanks.